setup.cfg

```diff
@@ -93,34 +94,36 @@ http2 =
 contextvars =
     contextvars >= 2.4, < 3; python_version < "3.7"
 
-all_non_platform =
-    %(test)s
+all =
+    pyobjc-core; platform_system == "Darwin"
+    pyobjc-framework-CFNetwork; platform_system == "Darwin"
+    pyobjc-framework-Cocoa; platform_system == "Darwin"
+    pywin32 != 226; platform_system == "Windows"
     %(tls)s
     %(conch)s
     %(serial)s
     %(http2)s
     %(contextvars)s
 
+all_non_platform =
+    %(all)s
```

Comment:
Thanks for the update. I think that this is an important one.

I think that all_non_platform should be kept as it is (with test removed) and not install pywin32.

and maybe create something like `_platform_specific` and add it to the new `all` toge

```
all_non_platform =
  AS_BEFORE

_platform_specific =
    pyobjc-core; platform_system == "Darwin"
    pyobjc-framework-CFNetwork; platform_system == "Darwin"
    pyobjc-framework-Cocoa; platform_system == "Darwin"
    pywin32 != 226; platform_system == "Windows"

all =
    %(all_non_platform)s
    %(_platform_specifics)s
```

Does it make sense?

The concern raised by Thomas is that maybe you are already calling `pip twisted[all_non_platform]` on pypy on windows or macos and you might end up with a failure.

---------

We don't have pypy test runs on macOS or Windows (I think that we should have at least one) so I don't know if that is the case.

But I think that is best to play it safe and keep `all_non_platform` is it was before.

Then we can add `all` or `standard` or any new extra :)

Reply:
Sorry for the late reply, do you think it would be preferable to use the `macos_platform` and `macos_platform` extra name or `_platform_specific` should be the way to go?

> [installing] on pypy on windows or macos and you might end up with a failure.

Yup pyobjc and pywin32 looks like CPython extension modules, I think I'll go head and add `platform_python_implementation == "CPython"` to the marker.